pkg/storage/gcp/bucket.go

```diff
@@ -0,0 +1,21 @@
+package gcp
+
+import (
+	"context"
+	"io"
+)
+
+// Bucket defines the storage bucket TODO
```

Comment:
maybe a little explanation about what a bucket means in GCP terms would be great for documentation :)